docs/register-create-stake-phase-2.md

```diff
@@ -0,0 +1,227 @@
+### 3.1. Registering a phase-1 stake
+
+To register a Bitcoin stake that is already on Bitcoin to 
+the Babylon chain (e.g., a phase-1 stake), you need to submit a 
+`MsgCreateBTCDelegation` with a valid inclusion proof. This requires your 
+staking transaction to already be confirmed on the Bitcoin network. You can do 
+this either manually through the Bitcoin network or using the staker library.
+
+1. Create `MsgCreateBTCDelegation` with the inclusion proof filled as defined 
+above and submit `MsgCreateBTCDelegation` to the Babylon network's btcstaking 
+module. You will then wait for the covenant committee to submit their signatures.
+
+
+Alternatively, you can use the CLI to register phase-1 stake:
+
+```bash
+babylond tx btcstaking create-btc-delegation [btc_pk] [pop_hex] [staking_tx] [fp_pk] [staking_time] [staking_value] [slashing_tx] [delegator_slashing_sig] [unbonding_tx] [unbonding_slashing_tx] [unbonding_time] [unbonding_value] [delegator_unbonding_slashing_sig]
+```
+
+2. You can check the status of your delegation by querying the babylon node 
+like this.
+
+```bash
+babylond query btcstaking btc-delegations <staking-tx-hash>
+```
+
+The above command will return the status of your delegation (pending, active, 
+unbonding, unbonded, any).
+
+### 3.2. Creating new stakes
+
+To create a new stake, you need to submit a `MsgCreateBTCDelegation` with an 
+empty inclusion proof to the Babylon network. The message should include:
+
+- Your unsigned staking transaction
+- Pre-signed slashing transaction
+- Unsigned unbonding transaction
+- Pre-signed unbonding slashing transaction
+- Proof of Possession (PoP)
+- Other required fields (staking amount, timelock periods, etc.)
+
+Please note that this flow can also be referred to as the Expression of Interest 
+(EOI) workflow. More information can be found [here](./staking-script.md).
+
+
+If you would like to create a new stake using the CLI, you can use the 
+following command. Please note that the `inclusion_proof` is optional and 
+can be left empty.
+
+```bash
+babylond tx btcstaking create-btc-delegation [btc_pk] [pop_hex] [staking_tx] [inclusion_proof] [fp_pk] [staking_time] [staking_value] [slashing_tx] [delegator_slashing_sig] [unbonding_tx] [unbonding_slashing_tx] [unbonding_time] [unbonding_value] [delegator_unbonding_slashing_sig]
+```
+
+2. Wait for covenant signatures to be collected. The covenant committee will 
+verify your transactions and add their required signatures. This ensures your 
+delegation will be accepted once you commit your BTC.
+
+3. Once you have covenant approval, the vigilante service will send the staking 
+transaction to the Bitcoin network. If you wish to submit the inclusion proof 
+manually, you can do so via `MsgAddBTCDelegationInclusionProof`. While 
+our Vigilante service will automatically handle this, you can also submit it 
+yourself.
+
+To submit the inclusion proof manually, you can use the following command:
```

Comment:
We don't need to provide instructions on how to submit the inclusion proof. We can just detail the contents of the structure and how it can be filled and say that it can be broadcast using similar methods as with the create delegation message